src/pkg/packager/common.go

```diff
@@ -507,6 +505,12 @@ func (p *Packager) validateLastNonBreakingVersion() (err error) {
 		return nil
 	}
 
+	if cliVersion == "unset" {
+		unsetWarning := fmt.Sprintf(lang.CmdPackageDeployUnsetCLIVersionWarn, config.CLIVersion)
+		p.warnings = append(p.warnings, unsetWarning)
+		return nil
+	}
+
 	lastNonBreakingSemVer, err := semver.NewVersion(lastNonBreakingVersion)
```

Comment:
would rather have this logic in what is now line 521 instead of returning an error there.  This could also be `UnknownVersion` in addition to unset and they might also set it to a bad semantic version which we might not want to error out for.

Reply:
Changes made